solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -1258,9 +1270,7 @@ contract LiquidityProtection is TokenHandler, ContractRegistryClient, Reentrancy
         uint256 _averageRateD,
         uint32 _maxDeviation
     ) internal pure returns (bool) {
-        uint256 min = _spotRateN.mul(_averageRateD).mul(PPM_RESOLUTION - _maxDeviation).mul(
-            PPM_RESOLUTION - _maxDeviation
-        );
+        uint256 min = _spotRateN.mul(_averageRateD).mul(PPM_RESOLUTION - _maxDeviation).mul(PPM_RESOLUTION - _maxDeviation);
```

Comment:
This is a prettier change, so it'll revert later when someone uses formating.

Reply:
Yup, so it turns out that the prettier doesn't always actually prettifies things.
IMO, the previous (red) form makes it harder to read and comprehend the function, which is somewhat arithmetically complicated. I believe that it is better in the new (green) form.
Is there anything that we can do in order to tell it to skip this one?

Reply:
@barakman we'll handle prettier rules when we have some time to breath, it's a simple change that will auto-format so no need to worry about it